### ISSUE TITLE
PDF version

### ISSUE THREAD

**Issue by scls19fr** [chrisvoncsefalvay/learn-julia-the-hard-way#30]
Hello,

a PDF version will be a great feature, you can use pandoc... but you can also try https://www.gitbook.com/ which is open source also https://github.com/GitbookIO/gitbook

Kind regards

**Comment by chrisvoncsefalvay** [comment]
Good idea, I'll probably hammer together a Makefile sometime. I'm sorry, most of my strength goes towards developing the book _Learn Julia_ for Manning, but I do intend to keep this going.

**Comment by scls19fr** [comment]
HTML version is available here
https://www.gitbook.com/book/scls/ljthw/details

PDF version
https://www.gitbook.com/download/pdf/book/scls/ljthw

EPUB version
https://www.gitbook.com/download/epub/book/scls/ljthw

MOBI version
https://www.gitbook.com/download/mobi/book/scls/ljthw

**Comment by chrisvoncsefalvay** [comment]
Not bad!  Not sure what I think of someone else's name on the cover - this is a community project and ideally I wouldn't have anyone's there. I'll be working on profiles etc.

**Comment by scls19fr** [comment]
I will remove my version when you will post links of your version.

I have been looking at
https://www.gitbook.com/@chrisvoncsefalvay
but it doesn't seems to be your username on GitBook

Compiling remotely with GitBook always use GitBook user name as author name
 (so your name will appear if you don't have a title page set when you will set GitBook).

Even setting author name in `book.json` doesn't fix this https://github.com/GitbookIO/gitbook/issues/649

**Comment by scls19fr** [comment]
I've just found https://www.gitbook.com/@octowombat

You just need to create a new book on GitBook and link it to this repository
https://github.com/chrisvoncsefalvay/learn-julia-the-hard-way

An other approach might be to turn to a GitHub organization.
So the name of the organization will (should) appear on title page
